src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -12,7 +12,8 @@
 
 import rbd
 
-from . import ApiController, RESTController, Task, UpdatePermission, DeletePermission
+from . import ApiController, RESTController, Task
+from . import UpdatePermission, DeletePermission, CreatePermission
```

Comment:
please move these imports to the previous line and use the `\` character to break the line. Check other files for examples.

Reply:
Thanks for the tip! Have done this now.